livekit-plugins/livekit-plugins-simli/livekit/plugins/simli/avatar.py

```diff
@@ -40,8 +40,8 @@ class SimliConfig:
             Maximum duration the avatar is not speaking for before the avatar disconnects.
     """
 
-    api_key: str
-    face_id: str
+    api_key: str = os.getenv("SIMLI_API_KEY")
+    face_id: str = os.getenv("SIMLI_FACE_ID")
```

Comment:
```suggestion
    api_key: NotGivenOr[str] = NOT_GIVEN
    face_id: NotGivenOr[str] = NOT_GIVEN
```
and read the value from env if not given in AvatarSession ctor or start?